From 8d6b124f2a0e4ab4e3d61415cec77038b94e517f Mon Sep 17 00:00:00 2001
From: Hans de Goede <hdegoede@redhat.com>
Date: Tue, 9 Nov 2010 11:32:07 +0100
Subject: [PATCH spice 2/7] spicec: Remove empty show / hide gui functions

When compiling without gui support just don't call show / hide
gui, rather then making them stubs, this makes it easier to follow what is
going on.
---
 client/application.cpp |    4 ++++
 client/application.h   |    4 ----
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/client/application.cpp b/client/application.cpp
index 7ef2c78..2701641 100644
--- a/client/application.cpp
+++ b/client/application.cpp
@@ -841,7 +841,9 @@ void Application::on_disconnected(int error_code)
 
     // todo: display special notification for host switch (during migration)
     set_state(DISCONNECTED);
+#ifdef USE_GUI
     show_gui();
+#endif
     if (host_switch) {
         _client.connect(true);
     }
@@ -853,7 +855,9 @@ void Application::on_visibility_start(int screen_id)
         return;
     }
     set_state(VISIBILITY);
+#ifdef USE_GUI
     hide_gui();
+#endif
     show_info_layer();
 }
 
diff --git a/client/application.h b/client/application.h
index 19c68a5..0e761ec 100644
--- a/client/application.h
+++ b/client/application.h
@@ -322,10 +322,6 @@ private:
     void create_gui_barrier(RedScreen& screen, int id);
     void destroyed_gui_barrier(int id);
     void destroyed_gui_barriers();
-#else // USE_GUI
-    void show_gui() {}
-    void hide_gui() {}
-
 #endif // USE_GUI
 
     // returns the press value before operation (i.e., if it was already pressed)
-- 
1.7.3.2

